src/components/WatchNow/styles.scss

```
@@ -0,0 +1,25 @@
+// Mobile portrait
+.watch {
+	display: flex;
+	position: absolute;
+	width: 11em;
+	height: 3.45em;
+	bottom: 5%;
+	left: 50%;
+	transform: translateX(-50%);
+	z-index: 3;
```

```suggestion
	z-index: 4;
```
Otherwise the button looks like it's behind the ghosts on the Episodes page.